cliquet_docs/reference/notifications.rst

```diff
@@ -3,30 +3,64 @@
 Notifications
 #############
 
-Knowing when a bucket or a collection has been modified is very
+Knowing collection has been modified is very
```

Comment:
s/collection/a collection/

Also, this event is triggered if we change a bucket, no ?